packages/icon-builder/test/index.js

```diff
@@ -120,14 +118,14 @@ describe('--svg-dir, --innerPath, --fileSuffix', function() {
   });
 
   after(function() {
-    temp.cleanupSync();
+    // temp.cleanupSync();
   });
 
   it('script outputs to directory', function(done) {
     builder.main(options, function() {
       assert.ok(fs.lstatSync(tempPath).isDirectory());
```

Comment:
I hate `assert.ok`. I don't think that *truthy* is good enough for tests. We need precision.
If you don't mind:
```js
assert.strictEqual(fs.lstatSync(tempPath).isDirectory(), true);
```